chore: v1.0.0-beta.127 (#1088)
Redocly/redocly-cli: docs/changelog.md

# Redocly CLI changelog

## 1.0.0-beta.127 (2023-05-?)

### Features

> - Display the update banner if a new version is available.

Reviewer: ```suggestion
- Display an update call-to-action if a new version is available.
```

> - Display an update call-to-action if a new version is available.

Reviewer 2: Could we just say "inform the user if a new version is available" please. "Call to action" feels like internal marketing language (also this sentence doesn't make sense as it is)

Author: @adamaltman  I see this is your change, what do you think? 
```suggestion
- Inform the user if a new version is available.
```

### Fixes

- Improved messages on wrong schemas.
- Fixed components duplication and self-referencing when bundling.

### Changes

> - New CLI arguments behaviour for array-like options: now it's required to specify the option multiple times to add multiple values. For example, `--skip-rule rule1 --skip-rule rule2` instead of `--skip-rule rule1 rule2`.

Reviewer: ```suggestion
- New CLI arguments behavior for array-like options: now it's required to specify the option multiple times to add multiple values. For example, `--skip-rule rule1 --skip-rule rule2` instead of `--skip-rule rule1 rule2`.
```
